src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestedCashAdvanceServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kuali.kfs.krad.service.BusinessObjectService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurPropertyConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestedCashAdvance;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestedCashAdvanceService;
+
+public class ConcurRequestedCashAdvanceServiceImpl implements ConcurRequestedCashAdvanceService {
+    protected BusinessObjectService businessObjectService;
+
+    @Override
+    public void saveConcurRequestedCashAdvance(ConcurRequestedCashAdvance concurRequestedCashAdvance) {
+        businessObjectService.save(concurRequestedCashAdvance);
+        
+    }
+
+    @Override
+    public boolean isDuplicateConcurRequestCashAdvance(ConcurRequestedCashAdvance concurRequestedCashAdvance) {
+        Collection<ConcurRequestedCashAdvance> concurRequestedCashAdvances = null;
+        Map<String, String> fieldValues = new HashMap<String, String>();
+        
+        fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.EMPLOYEE_ID, concurRequestedCashAdvance.getEmployeeId());
+        fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.REQUEST_ID, concurRequestedCashAdvance.getRequestId());
+        fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.PAYMENT_AMOUNT, concurRequestedCashAdvance.getPaymentAmount().toString());
+        
+        concurRequestedCashAdvances = businessObjectService.findMatching(ConcurRequestedCashAdvance.class, fieldValues);
+        
+        if ((concurRequestedCashAdvances == null) || (concurRequestedCashAdvances.isEmpty()) ){
```

Comment:
The conditional is looking for when there is nothing returned to indicate there is no duplicate. 
I thought having an explicit conditional statement was clearer than negating the result of the conditional.